infra/Pos/Communication/Relay/Logic.hs

```diff
@@ -452,9 +422,9 @@ invReqDataFlowNeighbors
        , Bi (InvOrData key contents)
        , Bi (ReqMsg key)
        )
-    => Text -> SendActions m -> key -> contents -> m ()
-invReqDataFlowNeighbors what sendActions key dt = handleAll handleE $
-    converseToNeighbors sendActions (pure . Conversation . invReqDataFlowDo what key dt )
+    => Text -> SendActions m -> MsgType -> key -> contents -> m ()
+invReqDataFlowNeighbors what sendActions msgType key dt = handleAll handleE $
+    void $ converseToNeighbors sendActions (sendMsg msgType) (pure . Conversation . invReqDataFlowDo what key dt) >>= waitForConversations
```

Comment:
Usually I am quite permissive with regards to long lines, but this one is 138 characters long, while [our style guide](https://github.com/input-output-hk/cardano-sl-style-guides/blob/master/haskell-style-guide.md#line-length) permits only 80 (100 in exceptional cases). It's really very long.